nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -199,8 +200,8 @@ class MegatronGPTModel(MegatronBaseModel, TextGeneration):
 
     def __init__(self, cfg: DictConfig, trainer: Trainer):
         if not HAVE_APEX:
-            raise ImportError(
-                "Apex was not found. Please see the NeMo README for installation instructions: https://github.com/NVIDIA/NeMo#megatron-gpt."
+            logging.warning(
+                "Only support running single gpu inference when Apex was not found. Please see the NeMo README for installation instructions: https://github.com/NVIDIA/NeMo#megatron-gpt."
```

Comment:
@ericharper  Apex isnt needed for multi GPU inference either right? Its only necessary (well, its recommended) for training right due to the fused distributed Adam?